File: GullSharksWeb/src/models/ShippingAddress.ts
export interface ShippingAddress {
  id: number;
  user_ID: number;
  city: string;
  country_ID: number;
  deliveryInstructions: string;
  province_ID: number;
  postalCode: string;
  streetAddress: string;
  isDeleted: boolean;
}

File: GullSharksLib/Models/ShippingAddress.cs
namespace GullSharksLib.Models;

public class ShippingAddress
{
    public int ID { get; set; }
    public int UserDetails_ID { get; set; }
    public string City { get; set; }
    public int Country_ID { get; set; }
    public int? Province_ID { get; set; }
    public string? PostalCode { get; set; }
    public string StreetAddress { get; set; }
    public string? DeliveryInstructions { get; set; }

    public bool IsDeleted { get; set; }
}

